packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -4236,6 +4236,51 @@ void main() {
     final TextField textField = tester.firstWidget(find.byType(TextField));
     expect(textField.restorationId, restorationId);
   });
+
+  testWidgets(
+    'DropdownMenu does not include the default trailing icon when showTrailingIcon is false',
+    (WidgetTester tester) async {
+      final ThemeData themeData = ThemeData();
+      await tester.pumpWidget(
+        MaterialApp(
+          theme: themeData,
```

Comment:
nit: seems themeData is not used anywhere, so we can remove this line and line 4243